web-ui/app/templates/compose/compose_box.hbs

```diff
@@ -1,8 +1,14 @@
 <button class="close-mail-button">
   <i class="fa fa-times"></i>
 </button>
-<input type="text" id="subject" value="{{subject}}" placeholder="{{t 'Subject'}}" tabindex="1"/>
-<textarea id="text-box" placeholder="{{t 'Body'}}" tabindex="2">{{body}}</textarea>
+<div class="compose">
```

Comment:
@dudadornelles i think it isn't, there's also `buttons-group columns` and a `clearfix` down there \/\/\/